src/status_im2/core.cljs

```diff
@@ -1,9 +1,5 @@
 (ns status-im2.core
   (:require
-    ;; NOTE: Do NOT sort i18n-resources because it MUST be loaded first.
-    [status-im2.setup.i18n-resources :as i18n-resources]
```

Comment:
hm

Reply:
seems like ignoring doesn't work? cc @yqrashawn

Reply:
the `clj-kondo/ignore` here is just telling `clj-kondo` don't print error message for this

it's the exclude regex in `make lint-fix` 
`clojure-lsp --ns-exclude-regex ".*/src/status_im2/core\.cljs$$" clean-ns` does the trick and it's working on my side

although it won't work in my editor since the exclude regex is not configured in my editor, maybe that's the reason?